### DISCUSSION TITLE
Pylontech US3000/US2000C

### DISCUSSION THREAD

**Discussion by gio-dot** [syssi/esphome-jk-bms#242] [Q&A]
Hi, i don't know and i never used JK BMS. I would know if the protocol you used for JK bms is similar/equal to that used in Pylontech batteries. Thank you
Myneed would be to communicate directly with an US3000C Pylontech batterie.

**Comment by syssi** [comment]
You should use this project instead: https://github.com/syssi/esphome-seplos-bms

The JK-BMS protocol has nothing in common with the seplos/pylontech protocol.

**Reply by gio-dot** [reply]
> You should use this project instead: https://github.com/syssi/esphome-seplos-bms
> 
> The JK-BMS protocol has nothing in common with the seplos/pylontech protocol.

Ok, i will look there. Shortly.. do you mean seplos protocol (that i don't know) is the same as Pylontech protocol?

**Reply by syssi** [reply]
It's not the same but very similar. The pylontech firmware supports some more registers/commands and some other requests get rejected (`NACK`). Feel free to create issues if you miss something.
